homeassistant/components/tessie/sensor.py

```diff
@@ -24,20 +25,31 @@
     UnitOfTemperature,
     UnitOfTime,
 )
-from homeassistant.core import HomeAssistant
+from homeassistant.core import HomeAssistant, callback
 from homeassistant.helpers.entity_platform import AddEntitiesCallback
 from homeassistant.helpers.typing import StateType
+from homeassistant.util import dt as dt_util
 
 from .const import DOMAIN
 from .coordinator import TessieStateUpdateCoordinator
 from .entity import TessieEntity
 
 
+@callback
+def hours_to_datetime(value: StateType) -> datetime | None:
+    """Convert relative hours into absolute datetime."""
+    return (
+        dt_util.now() + timedelta(hours=value)
+        if isinstance(value, (int, float)) and value > 0
+        else None
+    )
```

Comment:
```suggestion
    if isinstance(value, (int, float)) and value > 0:
        return dt_util.now() + timedelta(hours=value)
    return None
```
Please avoid multiline ternary operators

Reply:
That breaks the code coverage, I'm going to add tests and possibly a fixture just to meet this code formatting requirement. 😬